content/participate/index.html.haml

```diff
@@ -1,211 +1,225 @@
-    Advocacy and Outreach SIG
-  channels if you have any questions about contributing to Jenkins.
-  Any feedback about the guidelines will be appreciated.
+  - --
+  layout: simplepage
+  title: Participate and Contribute
+  section: participate
+  - --
+  %css
+    \.grid-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(256px, 1fr));
+    grid-gap: 20px;
+    margin-bottom: 1rem;
+    }
+    
+    \.channels-help {
+    text-align: center;
+    }
+    
+    \.card {
+    border: 1px solid #d3d3d3;
+    border-radius: .25rem;
+    }
+    
+    \.bg-img {
+    background-size: cover;
+    min-height: 20px;
+    background-position: center;
+    }
+    
+    \.content {
+    padding: 15px;
+    }
+    
+    \.bg-img-meet {
+    background-color: #A4262C;
+    }
+    
+    \.btn-meet {
+    border: 1px solid#A4262C;
+    color:
+    }
+    
+    \.btn-meet:hover {
+    background-color: #A4262C;
+    color: white;
+    }
+    
+    \.bg-img-code {
+    background-color: #40587C;
+    }
+    
+    \.btn-code {
+    border: 1px solid #40587C;
+    }
+    
+    \.btn-code:hover {
+    background-color: #40587C;
+    color: white;
+    }
+    
+    
+    \.bg-img-help {
+    background-color: #407855;
+    }
+    
+    \.btn-help {
+    border: 1px solid #407855;
+    }
+    
+    \.btn-help:hover {
+    background-color: #407855;
+    color: white;
+    }
+    
+    
+    \.bg-img-translate {
+    background-color: #8E6F33;
+    }
+    
+    \.btn-translate {
+    border: 1px solid #8E6F33;
+    }
+    
+    \.btn-translate:hover {
+    background-color: #8E6F33;
+    color: white;
+    }
+    
+    
+    \.bg-img-document {
+    background-color: #0078D4;
+    }
+    
+    \.btn-document {
+    border: 1px solid#0078D4;
+    }
+    
+    \.btn-document:hover {
+    background-color: #0078D4;
+    color: white;
+    }
+    
+    \.bg-img-design {
+    background-color: #038387;
+    }
+    
+    \.btn-design {
+    border: 1px solid #038387;
+    }
+    
+    \.btn-design:hover {
+    background-color: #038387;
+    color: white;
+    }
+    
+    
+    \.bg-img-test {
+    background-color: #867365;
+    }
+    
+    \.btn-test {
+    border: 1px solid #867365;
+    }
+    
+    \.btn-test:hover {
+    background-color: #867365;
+    color: white;
+    }
+    
+    
+    \.bg-img-review {
+    background-color: #CA5010;
+    }
+    
+    \.btn-review {
+    border: 1px solid #CA5010;
+    }
+    
+    \.btn-review:hover {
+    background-color: #CA5010;
+    color: white;
+    }
+    
+    h3 {
+    margin: 0;
+    }
+  %p
+    There are many ways to engage with the Jenkins project and community. Here are the most common ways to get you
+    started. Welcome aboard!
+  .grid-container
+    .card
+      .bg-img.bg-img-meet
+      .content
+        %h3 MEET
+        %p Meet other Jenkins users and share your experiences by organizing and attending events and meetups.
+        %br/
```

Comment:
Please use  CSS  instead of these BR tags (e.g. set `margin-bottom` for the `.card .content` selector. Also the nesting  change seems unnecessary (the link button can come after content, not be part of it).